README.md

```diff
@@ -0,0 +1,35 @@
+# go-mapbox-api
+
+API Wrapper for [Mapbox API](https://docs.mapbox.com/api/)
+
+## Usage
+
+### Initialization
+```go
+mapboxClient, err := NewClient(&MapboxConfig{
```

Comment:
We should move all of these examples to godoc so they don't become stale: https://godoc.org/github.com/fluhus/godoc-tricks#Examples

This readme should just be highlevel and link into godoc for any code or documentation

Reply:
We can wait to do this one until after we test out the API more